Form-Controls/styles.css

```diff
@@ -0,0 +1,114 @@
+/* General form styling */
+body {
+    font-family: Arial, sans-serif;
+    background-color: #f4f4f4;
+    margin: 0;
+    padding: 0;
+    color: #333; /* Dark grey for text */
```

Comment:
Is there another way to capture your definition of dark grey? What about a CSS variable?